### DISCUSSION TITLE
`Subscribe` button is absent

### DISCUSSION THREAD

**Discussion by kuraga** [community/community#123599] [Mobile]
### Select Topic Area

Bug

### Body

On every issue/PR
(UPD: in the repositories I `Watch` in some way),
under the kebab menu,
there is the `Unsubscribe` item/button.
It unsubscribes.

But there is no the `Subscribe`.

GitHub Mobile v1.161.0 (786), Android 14.

**Comment by mattruggio** [comment]
Hey @kuraga, thanks for the feedback! You should only see a `Subscribe` option in the kebab menu for issues and pull requests which you are currently _not_ subscribed to. If you are subscribed then you should only see an `Unsubscribe` option in the kebab menu.

There is a known bug (already logged internally) regarding [custom watch settings](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository) which may always show up as `Unsubscribe` but do nothing when clicked. Can you check on github.com website (not mobile app) to see if you may have [custom watch settings](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository) configured for the repository mentioned in regard to this bug?

**Reply by kuraga** [reply]
Hello @mattruggio ! Thanks for your reply. 

Seems like you're right, I'm subscribed to the Releases of that (buggy) repositories (sorry about incorrect `every`, just it's too often practice for me :smile:).

But... The `Unsubscribe` item really unsubscribes... on click (checked by a browser).The item's text instead is changed for a part of a second, only.

**Reply by mattruggio** [reply]
> The item's text instead is changed for a part of a second, only.

Thanks for clarifying your setup here. Let me add this to the notes of the internal bug, I have a feeling it is causing the issue here.

**Reply by kuraga** [reply]
@mattruggio , oh, seems like it got fixed?
